### DISCUSSION TITLE
Introduction of Github Co-Pilot and openAI Codex will inevitably lead to unparralleled prevalence of cheating.

### DISCUSSION THREAD

**Discussion by hm0809** [codewars/codewars.com#2734] [General]
Do not get me wrong, GitHub copilot is a truly remarkable integration and will certainly change the landscape of programming and software development forever. Unfortunately, for websites like Codewars, this means an artificially generated solution to even the hardest katas will be easier than ever to produce.

I am interested in what the discussion around this is. I assume the only option is to simply accept the fact that cheating is not preventable to any reliable degree and an 'honesty policy' will have to be indoctrinated. The rapid release of access to insanely powerful AI's was always going to happen at some point, the time happens to be now. Prepare. Adapt. Overcome.

**Comment by DonaldKellett** [comment]
Cheating might not be preventable, but we can simply punish cheaters as
they are discovered. So the accessibility of AI-based programming solutions
such as GitHub Copilot is no excuse to cheat ;-)

And no, solutions like GitHub Copilot are _not_ able to generate truly
unique code. So don't try your luck with it ;-)
